main.tf

```diff
@@ -108,6 +108,8 @@ resource "aws_lb_target_group" "default" {
   lifecycle {
     create_before_destroy = true
   }
+
+  tags = "${length(var.target_group_tags) == 0 ? module.default_target_group_label.tags : var.target_group_tags}"
```

Comment:
why not provide your custom tags in `var.tags`?
Those will be merged in `label` module with the  a few tags generated by the `label` module.
(unless you want a completely separate set of tags. What's the use case?)

Reply:
>why not provide your custom tags in var.tags?

The custom tags are specific to the particular target group and would not make sense being applied to the ALB resource. In my particular case, I want to use a terraform data source to find a target group by tags. Perhaps a compromise might be renaming this `target_group_additional_tags` and merging whatever is passed in with the label tags?

Reply:
yes, add `target_group_additional_tags` variable, then merge it with `label.tags` (not `var.tags`, those are included in the `label.tags` - and use the resulting map for the target group tags

Reply:
>add target_group_additional_tags variable, then merge it with label.tags 

done d5c7a49